File: Easy/FlipandInvertMatrix/FlipandInvertMatrix.py
class Solution:
    def solve(self, A):
        for i in range(len(A)):
            A[i].reverse()
            for j in range(len(A[i])):
                if A[i][j] == 0:
                    A[i][j] = 1
                else:
                    A[i][j] = 0
        return A

File: Easy/FlipandInvertMatrix/FlipandInvertMatrix.js
class Solution {
    solve(matrix) {
        for (let i = 0; i < matrix.length; i++) {
            matrix[i].reverse();
            for (let j = 0; j < matrix[i].length; j++) {
                matrix[i][j] = matrix[i][j] == 0 ? 1 : 0;
            }
        }
        return matrix;
    }
}